hw/src/scss/_weatherWidget.scss

```diff
@@ -0,0 +1,89 @@
+@import "_variablesForWW";
+@font-face {
+    font-family: $FONTF;
+    src: $FONTFURL format("opentype");
+}
+
+@mixin bcAndTs ($color, $tsColor, $tsSize: $SHODOWSIZE, $transparency:$SHODOWTRANSPARENCY) {
+    background-color: $color;
+    text-shadow: $tsSize rgba($tsColor, $transparency);
+}
+
+.weatherWidget {
+    font-family: $FONTF;
+    font-size: $FONTSMEDIUM;
+    line-height: $LINEH;
+    font-weight: bold;
+    text-transform: uppercase;
+    color: $WHITE;
+    height: $HEIGHTWW;
+    width: $WIDTHWW;
+    display: flex;
+    flex-flow: row wrap;
+    box-shadow: $BOXSHADOWWW;
+    position: absolute;
+    margin: auto;
+    top: $TOPWW;
+    right: $RIGHTWW;
+    bottom: $BOTTOMWW;
+    left: $LEFTWW;
+    .block {
+        height: $HEIGHTBLOCK;
+        width: $WIDTHBLOCK;
+        box-shadow: $BOXSHADOWBLOCK;
+        display: flex;
+        align-items: center;
+        align-content: space-between;
+        flex-direction: column;
+        justify-content: center;
+        &.first {
```

Comment:
If we are following the BEM naming methodology, we should avoid using such logical constructions and instead apply our styles directly to the class. This will improve code readability and maintainability:
```
//HTML
<div class="weather-widget__block weather-widget__block_first"></div>

//CSS
.weather-widget__block {
  height: 206px;
  width: 205px;
...
}

.weather-widget__block_first {
  background-color: #05b3db;
...
}
```